conclusions/rq2.md

```diff
@@ -0,0 +1,13 @@
+## What can be learned from existing efforts to involve a community with software development?
+In short, there are a few key factors that can be identified in succesful software development communities:
+- a small core team that actively communicates about the current and desired state of the software;
+- use of GitHub (git, pull-requests, issue-tracker) or a similar system;
+- modular code with standartized documentation, easy set-up and tutorials;
+- use both peer reviews as formal testing;
```

Comment:
`use both peer reviews as formal testing` ==> `use  of both peer reviews as well as formal testing`